packages/graphql/src/services/liveQuizzes.ts

```
@@ -473,118 +520,251 @@ export async function manipulateLiveQuiz(
     },
   })
 
+  // make sure that every element could be found and create a map for efficient access
   const uniqueElements = new Set(dbElements.map((q) => q.id))
   if (dbElements.length !== uniqueElements.size) {
     throw new GraphQLError('Not all elements could be found')
   }
-
   const elementMap = dbElements.reduce<Record<number, Element>>(
     (acc, elem) => ({ ...acc, [elem.id]: elem }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)